bin/build

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
```

Comment:
`bin/build` seems like a silly name since it mostly doesn't build. `bin/dev-tools` maybe?